src/plugins/gui/providers/bityProvider.ts

```diff
@@ -18,7 +18,7 @@ import {
   FiatProviderQuote
 } from '../fiatProviderTypes'
 
-const pluginId = 'bity'
+const providerId = 'bity'
```

Comment:
How is `pluginId` overloaded here?

Reply:
The type calls for `pluginId`. If that's incorrect, we need to change that.

Reply:
Pre-fix, we were passing along the currency pluginId, i.e. 'bitcoin' when we want the provider's id. The 'pluginId' arguably isn't even a pluginId since this is a provider, but a goal of this PR is to reduce impact since it's for 3.7.